devlink_linux.go

```diff
@@ -165,3 +165,68 @@ func (h *Handle) DevLinkGetDeviceList() ([]*DevlinkDevice, error) {
 func DevLinkGetDeviceList() ([]*DevlinkDevice, error) {
 	return pkgHandle.DevLinkGetDeviceList()
 }
+
+func parseDevlinkDevice(msgs [][]byte) (*DevlinkDevice, error) {
+	m := msgs[0]
```

Comment:
@fcrisciani , I think len > 0 is not needed, because ParseRouteAttr() does the below check,

for len(b) >= unix.SizeofRtAttr

Thanks for the review.